ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/pageheader/v1/pageheader/_cq_editConfig.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we really need this edit config? It's supposed to come from wcm text component.